docs/pull_request_template.md

```diff
@@ -6,15 +6,10 @@
 
 If you're unsure about a box, leave it unchecked. A maintainer will help you.
 
-If a box isn't applicable, add an explanation in **bold**.
-For example: **(N/A: this is a bugfix, not a feature)**
-
 - [ ] I understand README.md. I also understand that acceptance of
   community PRs will be delayed until the test and CI systems are online.
-- [ ] If this is a feature addition, that feature has been voted into the
-  C++ Working Draft.
 - [ ] Identifiers in product code changes are properly `_Ugly` as per
-  https://eel.is/c++draft/lex.name#3.1 .
+  https://eel.is/c++draft/lex.name#3.1 or there are no product code changes.

```

Comment:
For the folks who don't understand why all unicorns are blue. (no change requested.)